.github/workflows/lint.yaml

```diff
@@ -18,18 +17,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v3
-        with:
-          enable-cache: true
-          cache-dependency-glob: pyproject.toml
+        uses: astral-sh/setup-uv@v4
 
       - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version-file: pyproject.toml
+        shell: bash
+        run: uv venv
 
       - name: Install dependencies
         run: uv pip install . ruff
 
       - name: Lint with ruff
-        run: ruff check --output-format=github .
+        run: uv run ruff check --output-format=github .
```

Comment:
**[Question, non-blocking]** This is totally unrelated to this PR, so no need to resolve now, but I wonder if we really need this `lint` workflow at all given that [pre-commit is running ruff anyway](https://github.com/ccao-data/assesspy/blob/5e309db1434de96821a384bc2e3e556ec5076563/.pre-commit-config.yaml#L2-L8)?

Reply:
Good thinking, another holdover from the R package. I removed it in 55e3225.